conf-sample.js

```diff
@@ -25,8 +26,8 @@ c.gdax.passphrase = 'YOUR-PASSPHRASE'
 
 // to enable Poloniex trading, enter your API credentials:
 c.poloniex = {}
-c.poloniex.key = 'YOUR-API-KEY'
-c.poloniex.secret = 'YOUR-SECRET'
+c.poloniex.key = ''
+c.poloniex.secret = ''
```

Comment:
Lazyness :( Sorry if all this is a lot of trouble, someone told me to make a PR when this should have probably remained a fork when I have no idea how well any of this will be compatible with other exchanges. Don't get me wrong I'd love to help you guys out. Just doing my best. I'll fix this.